src/app/app.component.html

```diff
@@ -1,3 +1,8 @@
 <div *ngIf="!loading" class="full-screen">
-  <router-outlet></router-outlet>
+  <div *ngIf="isCallbackValid">
+    <router-outlet></router-outlet>
+  </div>
+  <div *ngIf="!isCallbackValid">
```

Comment:
I don't like polluting the app component with error handling like this. Can we not worry about this here and just display an error in any component that requires a callback?